NDTensors/src/lib/GradedAxes/src/gradedunitrange.jl

```diff
@@ -68,6 +68,7 @@ end
 # == is just a range comparison that ignores labels. Need dedicated function to check equality.
 struct NoLabel end
 blocklabels(r::AbstractUnitRange) = Fill(NoLabel(), blocklength(r))
+blocklabels(la::LabelledUnitRange) = label(la)
```

Comment:
```suggestion
blocklabels(la::LabelledUnitRange) = [label(la)]
```
`blocklabels(r)` should output a collection of labels of length `blocklength(r)`.